example/webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = (env = {}) => {
     poolTimeout: env.watch ? Infinity : 2000,
   };
   const workerPoolSass = {
+    workerAllowedFunctions: ['cssLoaderGetLocalIdent'],
```

Comment:
Just idea - why do not try to make it for all functions, I understand that it will not work when it using something from global scope, but anyway it will solve other simple cases

Sorry for delay. What do you think?